src/reducers/index.js

```diff
@@ -43,6 +43,18 @@ const reducer = (state = defaultState, action) => {
     return Object.assign({}, state, modifiedState);
   }
 
+  case 'SET_STATE_VALUES': {
```

Comment:
Finally! I've been thinking of this ;)

`SET_STATE_VALUE` can be merged into this, but it's fine.